CHANGELOG.md

```diff
@@ -24,5 +24,6 @@ the [releases page](https://github.com/Consensys/teku/releases).
 - Added Deneb (aka Dencun) configuration for Chiado network for epoch 516608 (2024-01-31 18:15:40 UTC).
 - Added Deneb (aka Dencun) configuration for Holesky network for epoch 29696 (2024-02-07 11:34:24 UTC).
 - Generate key at `—p2p-private-key-file` path if specified file doesn't exist.
+- Added `--stop-vc-when-validator-slashed-enabled` option to stop the VC when a validator is slashed
```

Comment:
If i understand this PR correctly, this is not the final part of this feature... changelog may be premature.

Reply:
Actually the feature is now complete. There was a missing part (BN/VC running in a single process) and it's implemented now